auditlog/tests/test_auditlog.py

```diff
@@ -459,21 +455,17 @@ def setUpClass(cls):
         cls.users_to_exclude_ids = cls.user.id
 
         # creating auditlog.rule
-        cls.auditlog_rule = (
-            cls.env["auditlog.rule"]
-            .with_context(tracking_disable=True)
-            .create(
-                {
-                    "name": "testrule 01",
-                    "model_id": cls.contact_model_id,
-                    "log_read": True,
-                    "log_create": True,
-                    "log_write": True,
-                    "log_unlink": True,
-                    "log_type": "full",
-                    "capture_record": True,
-                }
-            )
+        cls.auditlog_rule = cls.create_rule(
+            {
+                "name": "testrule 01",
+                "model_id": cls.contact_model_id,
+                "log_read": True,
+                "log_create": True,
+                "log_write": True,
+                "log_unlink": True,
+                "log_type": "full",
+                "capture_record": True,
+            }
         )
```

Comment:
```suggestion
        cls.auditlog_rule = cls.create_rule("testrule 01", cls.contact_model_id, log_type="full")
```

Reply:
Given that `create_rule` is essentially a stand-in for
```python
    @api.model_create_multi
    def create(self, vals_list):
```
I would either prefer to leave it as is, or adopt a signature even more similar.

Reply:
Force pushing a small change to support multi-create:
```diff
-        cls.models |= {rule.model_id.model}
+        cls.models |= set(rule.model_id.mapped('model'))
```